api/routes/agent.py

```diff
@@ -129,3 +121,8 @@ def set_specific_model(agent_name):
             return("Can't load model " + request_data.get("modelName") + "for bot " + agent_name)
     else:
         return response_template(400, "A body is mandatory inside the request")
+
+@current_app.route("/nlu/agents/<agent_name>/intents/<intent>/recurse/<recurse>/export", methods=["GET"])
```

Comment:
remove recurse from path and use instead query param request with default value false